integration_tests/conftest.py

```diff
@@ -33,6 +34,18 @@ def kill_process(process: subprocess.Popen) -> None:
         pass
 
 
+def get_network_host():
+    hostname = socket.gethostname()
+    ip_address = socket.gethostbyname(hostname)
+    # windows return ip address else return 0.0.0.0
+    # windows doesn't support 0.0.0.0
+
+    if platform.system() == "Windows":
```

Comment:
We used something else in other places in the code to get the OS:
```
if sys.platform.startswith("win32")
```
Can we use the same thing in all places ? Probably replace the occurences of the line I wrote above by `if platform.system() == "Windows"` which feels more understandable.

Reply:
Oh great shout! I will change it there too